OptimizelySDK.DemoApp/README.md

```diff
@@ -0,0 +1,50 @@
+
+* View the [C# Getting Started Guide](https://developers.optimizely.com/x/solutions/sdks/getting-started/index.html?language=csharp)
+
+* View the reference [documentation](https://developers.optimizely.com/x/solutions/sdks/reference/index.html?language=csharp).
+
+* Latest [C# SDK](https://github.com/optimizely/csharp-sdk)
+
+## Demo App
+
+This example App simulates an online retailer testing the effects of sorting products by price vs name.
+
+Using the instructions below, you can run the app locally and mimic bucketing website visitors by selecting a users from "Select Visitor" page. For example, the user ID “Mike” would simulate a unique visitor and a specific variation would be selected for that unique visitor. The variation that is given to a specific unique visitor, such as "Mike", will be deterministic. This means as long as the experiment conditions remain the same, Mike will always get the same variation.
+ 
+<img src="https://github.com/optimizely/csharp-sdk/blob/master/OptimizelySDK.DemoApp/images/screenshot.png">
+
+### Deploying and Running the App
+
+1. Login or create an [Optimizely Account](https://app.optimizely.com/signin).
+2. Setup a C# project via the Optimizely dashboard. [Instructions](https://developers.optimizely.com/x/solutions/sdks/getting-started/?language=csharp)
+3. For the application to bucket users properly, the following values need to be defined in the experiment.
+	* Variation Keys: The experiment must contain at least two variations with keys "sort_by_price" and "sort_by_name".
+	* Event Key: The tracked event with key "AddToCart".
+4. Run the application OptimizelySDK.DemoApp.
+5. Click the "Demo App" link to go to the "Configuration" page.
+6. Provide ProjectId, ExperimentKey and EventKey values and hit Save to load the datafile of your project in ProjectConfigJson textbox. The Optimizely client is now initialized with your project configuration and is ready to serve the requests.
+7. You’re all set. Play around and view the experiment's results! 
+
+To better understand this experiment, we recommend that you select a few different visitors on the "Select Visitor" page and bucket them into variations and simulate a conversion event by clicking the "Buy Now" button on the "Shop" page. Within a few seconds, you should see the results populate on the Optimizely results page.
+
+The crux of this Full Stack experiment is bucketing users into variations and exposing them to different sorting functions. The SDK’s `Activate()` function will bucket users into a variation based on a user ID (internal id, user cookie, etc…).
+
+To actually test which sorting algorithm influences increased sales, we need to track the number of clicks on the Buy Now button. We can leverage the SDK's `Track()` function for that, passing it the event key which, in our case, is "AddToCart" and the user ID.
+
+Finally, the "Messages" page lists the logs generated by Optimizely SDK in chronological order.
```

Comment:
Add a screenshot of messages page

Reply:
Same here. Screenshots of Demo App is not required in my opinion.